pkg/genericapiserver/options/server_run_options.go

```diff
@@ -159,6 +159,7 @@ func NewServerRunOptions() *ServerRunOptions {
 func (o *ServerRunOptions) WithEtcdOptions() *ServerRunOptions {
 	o.StorageConfig = storagebackend.Config{
 		Prefix: DefaultEtcdPathPrefix,
+		Quorum: DefaultReadQuorumFlag,
```

Comment:
Well - I think it should be discussed if we go with 3.1 or stay with 3.0.*
@xiang90 - so from what I understand, quorum-read wasn't optimal before 3.1 (in particular in 2.* and 3.0, right?). So how much slower it was before 3.1?